examples/guided_onboarding_example/public/components/step_one.tsx

```diff
@@ -23,27 +23,25 @@ import {
   EuiFormRow,
 } from '@elastic/eui';
 
-import useObservable from 'react-use/lib/useObservable';
-
 import { GuidedOnboardingPluginStart } from '@kbn/guided-onboarding-plugin/public/types';
 
 interface GuidedOnboardingExampleAppDeps {
-  guidedOnboarding: GuidedOnboardingPluginStart;
+  guidedOnboarding?: GuidedOnboardingPluginStart;
 }
 
 export const StepOne = ({ guidedOnboarding }: GuidedOnboardingExampleAppDeps) => {
-  const { guidedOnboardingApi } = guidedOnboarding;
-
   const [isTourStepOpen, setIsTourStepOpen] = useState<boolean>(false);
   const [indexName, setIndexName] = useState('test1234');
 
-  const isTourActive = useObservable(
-    guidedOnboardingApi!.isGuideStepActive$('testGuide', 'step1'),
-    false
-  );
   useEffect(() => {
-    setIsTourStepOpen(isTourActive);
-  }, [isTourActive]);
+    const subscription = guidedOnboarding?.guidedOnboardingApi
```

Comment:
I noticed that we still have a couple of places in the examples plugin that still mention `observable`. One is in the component `main`, but this was even before this PR. And the other mention is in this component, I think we could re-phrase those texts.

Reply:
I updated the copy to say an observable subscription; but happy to change to something else if you think it's still confusing